newspapers.com.js

```diff
@@ -0,0 +1,104 @@
+function detectWeb(doc, url) {  
+	return "newspaperArticle";
+}
+function doWeb(doc, url) { 
+	var namespace = doc.documentElement.namespaceURI;
+	var nsResolver = namespace ? function(prefix) {
+		if (prefix == 'x') return namespace; else return null;
+	} : null;
+
+	var newItem = new Zotero.Item("newspaperArticle");
+	var metaArr = new Object();
+	var metaTags = doc.getElementsByTagName("meta");
+	for (var i = 0 ; i < metaTags.length ; i++) {
+		metaArr[metaTags[i].getAttribute("property")] = metaTags[i].getAttribute("content");
+	}
+	/*
+	<meta property="og:title" content="Oct 11 1939 Detroit Free Press bork - on Newspapers.com">
+	*/
+	newItem.title = metaArr["og:title"].replace(/(.+?) -( on)? Newspapers.com/, "$1");
+	newItem.url = metaArr["og:url"];
+	
+	if (newItem.title.indexOf('/') >= 0) {
+		var tokens = newItem.title.split("/");
+		var author = tokens[1];
+		newItem.title = tokens[0].trim();
+	    if (author.match("; ")) {
+	      var authors = author.split("; ");
+	      for (var i in authors) {
+	        newItem.creators.push(Zotero.Utilities.cleanAuthor(authors[i], "author"));
+	      }
+	    } else {
+	      newItem.creators.push(Zotero.Utilities.cleanAuthor(author, "author"));
+	    }
+	}
```

Comment:
It looks for me that all the information you need you will extract below from the `printlocation` below. Is there an example where you need these lines here?